packages/@okta/i18n/src/properties/login_ja.properties

```diff
@@ -192,7 +192,7 @@ errors.E0000110 = \u6709\u52B9\u671F\u9650\u304C\u5207\u308C\u3066\u3044\u308B\u
 errors.E0000111 = \u8AAD\u307F\u53D6\u308A\u5C02\u7528\u3067\u3042\u308B\u305F\u3081\u3001\u30AA\u30D6\u30B8\u30A7\u30AF\u30C8{0}\u306F\u5909\u66F4\u3067\u304D\u307E\u305B\u3093\u3002
 errors.E0000112 = \u3053\u306E\u30E6\u30FC\u30B6\u30FC\u306F\u307E\u3060\u30A2\u30AF\u30C6\u30A3\u30D6\u306A\u72B6\u614B\u3067\u3042\u308A\u3001\u66F4\u65B0\u3067\u304D\u307E\u305B\u3093\u3002\u6570\u5206\u7D4C\u3063\u3066\u304B\u3089\u518D\u5EA6\u304A\u8A66\u3057\u304F\u3060\u3055\u3044\u3002
 # {0} is a server-side error response, oftentimes coming directly from a vendor.
-errors.E0000113 = {0}\u3002
+errors.E0000113 = {0}.
```

Comment:
sad :(
Globalization team is figuring out how to stop this from happening. Unicode should not be changed, since it is not the same as a regular `.`